steward/tests/acceptance.rs

```diff
@@ -20,75 +20,561 @@
+
+    let config = StewardConfig {
+        keys: KeysConfig {
+            keystore: keystore_dir_string.clone(),
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: keystore_dir_string.clone(),
+        ..Default::default()
+    };
+
+    let config_file_path = key_dir.path().join("config.toml");
+    let config_string = toml::to_string(&config).expect("could not write config to TOML string");
+    fs::write(config_file_path.clone(), config_string).expect("could not write config file");
+
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "cosmos",
+            "delete",
+            "cosmoskey",
+        ])
         .capture_stdout()
         .run();
+    //check that command executes without error
+    cmd.wait().unwrap().expect_success();
+
+    assert_eq!(key_file_path.exists(), false);
+
+    Ok(())
+}
+
+/// Eth recover key test
+#[test]
+fn cosmos_keys_recover() -> io::Result<()> {
+    let mut runner: Lazy<CmdRunner> = Lazy::new(|| CmdRunner::default());
+    let key_dir = TempDir::new("test_key")?;
 
-    cmd.stdout().expect_line("Hello, acceptance test!");
+    let keystore_dir_path = key_dir.path().join("keystore");
+    let keystore_dir_string = keystore_dir_path
+        .clone()
+        .into_os_string()
+        .into_string()
+        .unwrap();
+    fs::create_dir(keystore_dir_path.clone()).expect("could not create keystore dir");
+
+    let key_file_path = keystore_dir_path.join("cosmkey.pem");
+
+    let config = StewardConfig {
+        keys: KeysConfig {
+            keystore: keystore_dir_string.clone(),
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: keystore_dir_string.clone(),
+        ..Default::default()
+    };
+
+    let config_file_path = key_dir.path().join("config.toml");
+    let config_string = toml::to_string(&config).expect("could not write config to TOML string");
+    fs::write(config_file_path.clone(), config_string).expect("could not write config file");
+
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "cosmos",
+            "recover",
+            "cosmkey",
+            "merry blush enforce kite tired cheap use learn there oyster possible thank woman web midnight december sentence ski throw father scheme element onion talent"
+        ])
+        .capture_stdout()
+        .run();
+    // Check that command executes without error.
     cmd.wait().unwrap().expect_success();
+    assert!(key_file_path.exists());
+
+    Ok(())
 }
 
-/// Example of a test which matches a regular expression
+/// Eth rename key test
 #[test]
-fn version_no_args() {
-    let mut runner = RUNNER.clone();
-    let mut cmd = runner.arg("--version").capture_stdout().run();
-    cmd.stdout().expect_regex(r"\A\w+ [\d\.\-]+\z");
+fn cosmos_keys_rename() -> io::Result<()> {
```

Comment:
Should there also be an `eth_key_rename` version of this test?  Also the comment above says "Eth" rather than "Cosmos".

Reply:
We don't have a rename command for Eth in Steward. Is it something you'd want me to add?

Reply:
We don't have a rename command for Eth in Steward. Is it something you'd want me to add?

Reply:
I just assumed both keys commands had the same functions. We might as well if there's a cosmos version.